dev-docs/src/theme/ReactLiveScope/index.js

```diff
@@ -7,10 +7,12 @@ let ExcalidrawComp = {};
 if (ExecutionEnvironment.canUseDOM) {
   ExcalidrawComp = require("@excalidraw/excalidraw");
 }
-const Excalidraw = React.forwardRef((props, ref) => {
+const Excalidraw = (props) => {
   const { colorMode } = useColorMode();
-  return <ExcalidrawComp.Excalidraw theme={colorMode} {...props} ref={ref} />;
-});
+  return (
+    <ExcalidrawComp.Excalidraw theme={colorMode} {...props} ref={props.ref} />
```

Comment:
Making Ref a prop is the best outcome of React 19 ❤️

Reply:
yeah, I realized it only too late. Will be reverting :)